proposals/lip-0051.md

```diff
@@ -1921,6 +1925,10 @@ def transferCrossChain(
         emitFailedCrossChainTransferEvent(senderAddress, tokenID, amount, receivingChainID, recipientAddress, INVALID_RECEIVING_CHAIN)
```

Comment:
This check should also be deleted now (and the `INVALID_RECEIVING_CHAIN` variable, since it would not be used anymore..)

Reply:
As discussed, it's not wrong to keep this check for reasons of clarity.